popup.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/css/popup.css" />
+    <title>Ghana Tech Summit</title>
+  </head>
+  <body>
+    <main>
+      <section class="homepage">
+        <header class="nav" id="header">
+          <div class="nav-container">
+            <a class="nav-logo" href="#">
+              <img src="./assets/logos/gts.png" alt="Ghana Tech Summit Logo" />
+            </a>
+            <div class="nav-social">
+              <div class="nav-flex">
+                <a href="#" target="blank">
+                  <img
+                    src="./assets/icons/twitter.png"
+                    alt="GTS Twitter Handle"
+                  />
+                </a>
+                <a href="#" target="blank">
+                  <img
+                    src="./assets/icons/facebook.png"
+                    alt="GTS Facebook Handle"
+                  />
+                </a>
+                <a href="#" target="blank">
+                  <h6>English</h6>
+                </a>
+                <a href="#" target="blank">
+                  <h6>MyPage</h6>
+                </a>
+                <a href="#" target="blank">
+                  <h6>Logout</h6>
+                </a>
+              </div>
+            </div>
+            <nav>
+              <img
+                src="./assets/icons/menu.svg"
+                class="nav-menu"
+                alt="Menu Options"
+              />
```

Comment:
- [ ] It looks like there is an issue with your JavaScript event for the hamburger menu on the about page. Right now, this event only works for the index page, so whenever I go to the about page and try to open it, it is not showing up. Take a look at your routing or any other possible issue so you can fix this.